vhdbuilder/packer/components.json

```diff
@@ -453,9 +453,8 @@
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
         "1.2.7",
-        "1.4.13",
-        "1.4.14",
-        "1.4.16"
+        "1.4.16",
```

Comment:
have to leave 1.4.14 since apparently thats what we're on?
https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=/toggles/global/azure-cni-linux-url.yaml

Reply:
and swift is on 1.4.12, so i have left 1.4.14 and 1.4.12 cached respectively.